tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -20,61 +20,63 @@ export const feature_mcolor3: Feature<string> = {
 
 export const flavor_text: Feature<string> = {
   name: 'Flavor Text',
-  description: 'Describe your character!',
+  description:
+    "Appears when your character is examined (but only if they're identifiable - try a gas mask).",
   component: FeatureTextInput,
 };
 
 export const silicon_flavor_text: Feature<string> = {
   name: 'Flavor Text (Silicon)',
-  description: 'Describe your cyborg/AI shell!',
+  description: "Only appears if you're playing as a borg/AI.",
   component: FeatureTextInput,
 };
 
 export const ooc_notes: Feature<string> = {
   name: 'OOC Notes',
-  description: 'Talk about your character OOCly!',
+  description:
+    'Primarily for things like ERP, general, antag/whatever preferences.',
   component: FeatureTextInput,
 };
 
 export const custom_species: Feature<string> = {
   name: 'Custom Species Name',
-  description:
-    "Want to have a fancy species name? Put it here, or leave it blank if you want to use your species' default name.",
```

Comment:
Why was this removed?

Reply:
irrelevant, this behavior is already inferred

Reply:
I don't agree, and the leaving it blank tip isn't always obvious